internal/exec/help.go

```diff
@@ -103,6 +103,19 @@ func processHelp(
 			"If the workspace does not exist, the command creates it by executing the 'terraform workspace new' command.\n\n" +
 			"Usage: atmos terraform workspace <component> -s <stack>\n\n" +
 			"For more details refer to https://atmos.tools/cli/commands/terraform/workspace\n")
+	} else if componentType == "workflow" && command == "" {
+		u.PrintMessage("\nAtmos workflow commands support failure handling and resume functionality:\n\n" +
+			"When a workflow step fails:\n" +
+			" - The failed step name and command will be displayed\n" +
+			" - A resume command will be provided to restart from the failed step\n\n" +
+			"Example:\n" +
+			"Step 'deploy-vpc' failed!\n" +
+			"Error: Error applying plan:\n" +
+			"1 error occurred: AWS API call failed\n\n" +
+			"Command failed: terraform apply vpc -auto-approve\n\n" +
+			"To resume the workflow from this step, run:\n" +
+			"atmos workflow deploy-infra -f workflow1 --from-step deploy-vpc\n\n" +
+			"For more details refer to https://atmos.tools/cli/commands/workflow/\n")
```

Comment:
Please share a screenshot of what this looks like

Reply:
Good point. Checking again, I noticed that this condition is never evaluated because we call the BOA (interactive menu) when the user executes `atmos workflow`. To display this message, I’ve moved it to the `atmos workflow --help` command. This is the final result, but it seems a bit lengthy to me. Let me know if you’d like it placed elsewhere or remove.

cc: @aknysh 
![CleanShot 2024-11-10 at 16 23 07](https://github.com/user-attachments/assets/2a7200a0-224c-466b-8fec-
7c6b7674494f)

Reply:
![CleanShot 2024-11-10 at 10 30 02](https://github.com/user-attachments/assets/c41203b0-82c0-4cfe-ad44-e28194c459d4)